specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -252,7 +252,9 @@ function TreeView<SCHEMA extends AnyTree>({
                   ${index === 0 ? '-ml-2 rounded-bl pl-4' : ''}
                   ${index + 1 === length ? '-mr-2 rounded-br pr-4' : ''}
                   ${
-                    reduceTransparency || !supportsBackdropBlur
+                    reduceTransparency ||
+                    !supportsBackdropBlur ||
+                    navigator.userAgent.includes('Firefox')
```

Comment:
This is not a correct fix.
This assumes that all versions of firefox don't support this feature and never will. It also assumes that all other browsers support that feature and always did.

Instead, we can specifically check if current browser supports a given CSS feature using the `@supports` keyword in CSS or `CSS.supports` function in javascript. See more - https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports_static

Please look at that article to familiarize yourself with how this works.
You don't have to actually write `CSS.supports` as it has already been done - that's what the `supportsBackdropBlur` variable right above this line is for.

In this case, `supportsBackdropBlur` happens to be true even in firefox, and yet blur doesn't work. Thus the next task here is to figure out why blur is not working (i.e, check other examples of backdrop blur where it works in firefox and see what are we doing differently)